wetmap/src/components/newModals/tripCreatorListModal/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { itineraries } from '../../../supabaseCalls/itinerarySupabaseCalls';
+import { updateDiveShop } from '../../../supabaseCalls/shopsSupabaseCalls';
+import { SelectedShopContext } from '../../contexts/selectedShopContext';
+import { UserProfileContext } from '../../contexts/userProfileContext';
+import { ModalContext } from '../../reusables/modal/context';
+import { ItineraryItem } from './types';
+import TripCreatorListView from './view';
+import { ModalHandleProps } from '../../reusables/modal/types';
+import DiveShopModal from '../shopModal/index';
+
+type TripCreatorListProps = Partial<ModalHandleProps>;
+
+export default function TripCreatorListModal(props: TripCreatorListProps) {
+  const { selectedShop } = useContext(SelectedShopContext);
+  const { profile } = useContext(UserProfileContext);
```

Comment:
not sure if we need to import the profile for this page, i think this can be removed